### ISSUE TITLE
Support historical pp versions?

### ISSUE THREAD

**Issue by jxu** [Francesco149/oppai-ng#75]
I think it could be very useful to support historical versions of the pp calculating system. That way it is possible to compare the pp values of pp records if they had been done at a certain point in time. What do you think?

**Comment by Francesco149** [comment]
yeah that would be cool, dont know when I will feel like going through all the changes history though
